docs/source/pyrsa.vis.mds_plot.rst

```diff
@@ -0,0 +1,7 @@
+pyrsa.vis.colors module
+=======================
+
+.. automodule:: pyrsa.vis.colors
```

Comment:
I think this should be `.. automodule:: pyrsa.vis.mds_plot`

Reply:
Oops, nice catch!. Just fixed it.